PWD_Vault/endecrypt.py

```diff
@@ -35,54 +36,63 @@ def PBKDF2(data, salt):
 def encrypt(key):
     fernet = Fernet(key)
 
-    with open('data.csv', 'rb') as file:
+    with open('PWD_Vault/PWD_Vault/data.csv', 'rb') as file:
         original = file.read()
 
     encrypted = fernet.encrypt(original)
 
-    with open('data.csv', 'wb') as encrypted_file:
+    with open('PWD_Vault/PWD_Vault/data.csv', 'wb') as encrypted_file:

```

Comment:
Why was this necessary?

Reply:
I just thought to maintain the format with others.